inc/fields/checkboxesfield.class.php

```diff
@@ -140,7 +140,7 @@ public function serializeValue() {
          return '';
       }
 
-      return Toolbox::addslashes_deep(json_encode($this->value, JSON_OBJECT_AS_ARRAY));
+      return Toolbox::addslashes_deep(json_encode($this->value, JSON_OBJECT_AS_ARRAY+JSON_UNESCAPED_UNICODE));
```

Comment:
this change fix unicode e"scaping for checkboxes and multiselect. Seel below : before and after

![image](https://user-images.githubusercontent.com/14139801/82725657-ac8fcf00-9cde-11ea-97f3-749650a603e9.png)